### DISCUSSION TITLE
memos-deployment/

### DISCUSSION THREAD

**Discussion by ghost** [eallion/eallion.com#106] [Giscus]
# memos-deployment/

这篇文章介绍了Memos简介及其部署方法。作者提到了他最早用的微博客程序是PageCookery，并介绍了“B言B语”和Memos的应用。文章还提供了Memos的常用命令和升级命令，提到如果要对互联网提供Memos访问服务，需要使用Nginx反代5230端口。最后，文章介绍了Memos的数据持久化配置。

https://www.eallion.com/memos-deployment/

**Comment by xinkeji** [comment]
你好蜗牛大佬，一直在关注着你，最近突然也想搞一个hugo，没事当空间发，想使用你的源码也做一个博客，然后也很感谢你开源的源码，然后因为我最主要的还是想要memos这个功能，但是不知道怎么去获取你那样的api链接，不自己搭建，准备直接用官方的，请问有办法嘛？毕竟不想为了上传图片还要去找第三方图床，emm这点我应该和大佬不同，在抒发感受的同事突然想着突然放第三方图床感觉确实有点打扰雅兴😂😂

**Reply by xinkeji** [reply]
我草，大佬你这邮箱自动回复的功能？？？？，我草

**Reply by eallion** [reply]
我现在用的是 Mastodon 了。你先看看 https://github.com/eallion/memos.top 和木木老师的 https://github.com/lmm214/memobbs 适不适合你。

**Reply by eallion** [reply]
非常感谢您的关注！

**Comment by refrain1128** [comment]
大佬，我该用mysql数据库使用一段时间后，发现所有账号登录不上了，显示user not found，注册也一直在转圈圈。这是啥情况啊？
设置的环境变量
MEMOS_DRIVER=mysql
MEMOS_DSN=memos:Me@52555555@tcp(172.17.0.1)/memos

**Reply by eallion** [reply]
我已经超过半年没用 Memos 了，现在的 Memos 只是备份 Mastodon 的内容的。不是很了解新版本的 Memos 喔。现在的版本更新也比较迷，你回滚到之前的版本试试看。（记得备份后再操作）

**Reply by refrain1128** [reply]
破案了，memos 0.20.0版本的问题，https登录没问题，http在非服务器设备登录不上去

**Reply by eallion** [reply]
这个问题还不容易排查，能找到问题挺好的。
